libbeat/tests/system/test_seccomp.py

```diff
@@ -0,0 +1,44 @@
+import platform
+import unittest
+from base import BaseTest
+
+
+def is_version_below(version, target):
+    t = map(int, target.split('.'))
+    v = map(int, version.split('.'))
+    v += [0] * (len(t) - len(v))
+    for i in range(len(t)):
+        if v[i] != t[i]:
+            return v[i] < t[i]
+    return False
+
+
+# Require Linux greater than 2.6.12 and 386/amd64 platform
+def is_seccomp_supported():
+    p = platform.platform().split('-')
+    if p[0] != 'Linux':
+        return False
+    if is_version_below(p[1], '2.6.12'):
```

Comment:
The requirement is 3.17 because we need `SECCOMP_FILTER_FLAG_TSYNC` (aka `seccomp.FilterFlagTSync`) to sync the policy to all of the threads Go is managing for us.